### PR TITLE
[IMP] pos_gift_card: Barcode usage

### PR DESCRIPTION
**Pull request by rhe-odoo**
Allow to use the bar code in the popup gift card properly.
Small change of the gift card code generation.
Add a missing label in the view for sale gift card and pos gift card

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by JKE-be** [comment]
n'hesite pas a ajouter un commentaire du coup au dessus du 044 car au prochain refactoring, qqun risque de le retirer par erreur ...

Sans connaitre ca https://github.com/odoo/odoo/pull/75937/files#diff-f3d727eff0920ae8ccb6b4011c172e2326f52a7586d4ea98c03266b4958b036bR24 c'est dur a comprendre ;)

merci pour les changements, et nice d'avoir réutilisé et rendu encore plus generique Gift card ;) nice job

File: addons/pos_gift_card/static/src/css/giftCard.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.giftCardPopupMain {
+.pos .popups  .giftCardPopupMain {
```
Comment (JKE-be) [inline addons/pos_gift_card/static/src/css/giftCard.css:1]: 2x ' '

**Comment by caburj** [comment]
I believe this fix should target saas-14.5. But if not, at least the access rights changes should be in saas-14.5.

**Comment by C3POdoo** [comment]
Dear @rhe-odoo,

Thank you for your contribution but we are closing it due to inactivity.
We only support the last 3 stable versions so no longer accepts patches into this branch.

We apology if we could not look at your request in time.
If the contribution still makes sense, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

_This is an automated message._

**Pull request closed without merging**